src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -29,7 +29,7 @@ public void AddTrace(string trace)
 
         internal LogLevel LogLevel { get; set; } = LogLevel.Info;
 
-        internal ConcurrentDictionary<string, int> UnfinishedFailedMessages = new ConcurrentDictionary<string, int>();
+        internal ConcurrentDictionary<string, bool> UnfinishedFailedMessages = new ConcurrentDictionary<string, bool>();
```

Comment:
I remember removing the subscription to the retries events, but are we sure that the bool flag is good enough instead of a counter?